examples/vite-app/src/main.ts

```diff
@@ -13,27 +12,24 @@ const messageHistory = document.createElement('div');
 container.appendChild(messageHistory);
 messageHistory.innerHTML = `<div style="margin-top:20px;">Message History:</div>`;
 
-const appendMessage = (message: JSONMessage | AudioMessage) => {
+const appendMessage = (message: Hume.empathicVoice.SubscribeEvent) => {
   const timestamp = new Date().toLocaleTimeString();
 
   const messageContainer = document.createElement('div');
-  if (message.type === 'assistant_message' || message.type === 'user_message') {
```

Comment:
accidental deletion?

Reply:
https://dev.hume.ai/reference/empathic-voice-interface-evi/chat/chat#send.User%20Input.type I think that `user_message` is a deprecated/legacy type (no longer in the API docs). 

@gregorybchris can you confirm?

Reply:
Thanks for catching!